tests/table/test_analytics_table.py

```diff
@@ -0,0 +1,422 @@
+        "event2": Event(source="event1", display_name="event1", window_hr=2, aggregation_method="min"),
+    }
+    mock_config.target = "event1"
+    mock_config.entity_keys = ["entity"]
+    mock_config.predict_time = "time"
+    mock_config.cohorts = [Cohort(source=name) for name in ["cohort1", "cohort2"]]
+    mock_config.features = ["one"]
+    mock_config.config_dir = tmp_path / "config"
+    mock_config.censor_min_count = 0
+    mock_config.targets = ["event1"]
+    mock_config.output_list = ["prediction"]
+
+    return mock_config
+
+
+def get_test_loader(config):
+    mock_loader = Mock(autospec=SeismogramLoader)
+    mock_loader.config = config
+
+    return mock_loader
+
+
+def get_test_data():
+    return pd.DataFrame(
+        {
+            "entity": ["A", "A", "B", "C"],
+            "prediction": [1, 2, 3, 4],
+            "time": ["2022-01-01", "2022-01-02", "2022-01-03", "2022-01-04"],
+            "event1_Value": [0, 1, 0, 1],
+            "event1_Time": ["2022-01-01", "2022-01-02", "2022-01-03", "2021-12-31"],
+            "event2_Value": [0, 1, 0, 1],
+            "event2_Time": ["2022-01-01", "2022-01-02", "2022-01-03", "2022-01-04"],
+            "cohort1": [1, 0, 1, 0],
+            "cohort2": [0, 1, 0, 1],
+        }
+    )
+
+
+@pytest.fixture
+def fake_seismo(tmp_path):
+    config = get_test_config(tmp_path)
+    loader = get_test_loader(config)
+    sg = Seismogram(config, loader)
+    sg.dataframe = get_test_data()
+    yield
+
+    Seismogram.kill()
+
+
+class TestPerformanceMetrics:
+    def test_initialization(self, fake_seismo):
+        df = pd.DataFrame(
+            {
+                "score1": [0.1, 0.4, 0.35, 0.8],
+                "score2": [0.2, 0.5, 0.3, 0.7],
+                "target1": [0, 1, 0, 1],
+                "target2": [1, 0, 1, 0],
+            }
+        )
+        statistics_data = pd.DataFrame(
+            {
+                "Score": ["score1", "score1", "score2", "score2"],
+                "Target": ["target1", "target2", "target1", "target2"],
+                "AUROC": [0.75, 0.85, 0.6, 0.5],
+                "PPV": [0.6, 0.7, 0.5, 0.3],
+            }
+        )
+        scores = ["score1", "score2"]
+        targets = ["target1", "target2"]
+        metric_values = [0.7, 0.8]
+        table_config = AnalyticsTableConfig(
+            columns_show_bar={"AUROC": "lightblue", "PPV": "lightgreen"},
+            columns_show_percentages=["Prevalence"],
+            color_bar_style=1,
+        )
+        pm = PerformanceMetrics(
+            df=df,
+            score_columns=scores,
+            target_columns=targets,
+            metric="sensitivity",
+            metric_values=metric_values,
+            statistics_data=statistics_data,
+            table_config=table_config,
+        )
+
+        assert pm.df.equals(df)
+        assert pm.score_columns == scores
+        assert pm.target_columns == targets
+        assert pm.metric == "Sensitivity"
```

Comment:
It seems like `assert pm.metric == "Sensitivity"` would fail here since pm.metric is init as "sensitivity". I haven't run pytest yet though

Reply:
When initializing pm.metric, the input is case insensitive (we use Metric enum class to transform it into standard format of "Sensitivity").